src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+to keep up with their progress, follow [Unlocking new capabilities for the
+web](https://developers.google.com/web/updates/capabilities).
+{% endAside %}
+
+## What is Web NFC? {: #what }
+
+Web NFC provides sites the ability to read and write to NFC tags when they are
+brought in close proximity to the user's device (usually 5-10 cm, 2-4 inches).
+The current scope is limited to NDEF, a lightweight binary message format that
+works across different tag formats.
+
+{% Aside 'gotchas' %}
+Web NFC is limited to NDEF because the security properties of reading and
+writing NDEF data are more easily quantifiable. Low-level I/O operations (e.g.
+ISO-DEP, NFC-A/B, NFC-F) and Host-based Card Emulation (HCE) are not supported.
+{% endAside %}
+
+<figure class="w-figure">
+  <img src="./nfc-operation-diagram.png" alt="Phone powering up an NFC tag to exchange data">
+  <figcaption class="w-figcaption">Diagram of an NFC operation</figcaption>
+</figure>
+
+## Suggested use cases {: #use-cases }
+
+Examples of sites that may use Web NFC include:
+- Museums and art galleries can display more information about a display when
+  the user touches their device to a NFC card near the exhibit.
+- Inventory management sites can read or write data to the NFC tag on a
+  container to update information on its contents.
+- Conference sites can use it to scan NFC badges during the event.
+- Sites can use it for sharing initial secrets needed for device or service
+  provisioning scenarios and also to deploy configuration data in operational
+  mode.
+
+<figure class="w-figure">
+  <img src="./nfc-inventory-management.png" alt="Phone scanning several NFC tags">
+  <figcaption class="w-figcaption">NFC inventory management illustrated</figcaption>
+</figure>
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                       | Status                       |
+| ------------------------------------------ | ---------------------------- |
+| 1. Create explainer                        | [Complete][explainer]        |
+| 2. Create initial draft of specification   | [Complete][spec]             |
+| **3. Gather feedback and iterate design**  | [**In Progress**](#feedback) |
+| **4. Origin trial**                        | [**In Progress**][ot]        |
+| 5. Launch                                  | Not Started                  |
+
+</div>
+
+## Using Web NFC {: #use }
+
+### Enabling support during the origin trial phase
+
+Web NFC will be available on Android as an origin trial in Chrome 81. The origin
+trial is expected to end in Chrome 84.
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Enabling via chrome://flags
+
+To experiment with Web NFC locally on Android, without an origin trial token,
+enable the `#enable-webnfc` flag in `chrome://flags`.
```

Comment:
Why not add the actual link :) ? chrome://flags/#enable-webnfc

Reply:
`chrome://` URLs are not linkable in web contents for security reasons in browsers.